processors/config/anaSvtBl2D_cfg.py

```diff
@@ -0,0 +1,44 @@
+import HpstrConf
+import sys
+import os
+
+# Use the input file to set the output file name
+infile = sys.argv[1].strip()
```

Comment:
Please change this to the established way of using -i and -o to specify the input and output filenames. This is ok for your own personal local configs, but should not be like this in the repo.